schema/src/main/jade-assets/clinvar_release.asset.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "clinvar_release",
+  "rootTable": "xml_archive",
+  "rootColumn": "release_date",
+  "tables": [
+    {
+      "name": "clinical_assertion",
+      "columns": []
```

Comment:
Is the empty set here just the default value (so that it defaults to using all of them)? Or is this actually specifying `[]` as the set of columns?

Reply:
The 1st option, it's the one shortcut they've built in so far